package.json

```diff
@@ -15,11 +15,14 @@
     "apm": "./bin/apm"
   },
   "scripts": {
-    "build": "node script/check-version.js && grunt",
-    "prepare": "grunt prepare",
+    "check-version": "node script/check-version.js",
+    "clean": "shx rm -rf lib bin/node_darwin_x64 && shx mkdir -p lib",
```

Comment:
I think the old Grunt "clean" task was really out-of-date. (This "node_darwin_x64" thing was added 7 years ago https://github.com/atom/apm/blame/v2.5.0/Gruntfile.coffee#L34). We never end up with anything called `bin/node_darwin_x64` anymore.

It's just `bin/node` now, on both Linux and macOS. And on Windows it's `bin/node.exe`.

Reply:
This is already fixed on master.

Reply:
I noticed that, thanks. But master still "deletes" `node_darwin_x64` which is never going to exist.